### ISSUE TITLE
Dependency error between flutter_driver and flutter_cache_manager on stable

### ISSUE THREAD

**Issue by swoon-lq** [Baseflow/flutter_cache_manager#293]
## Steps to Reproduce

1. Update to Flutter 2.
2. Add flutter_cache_manager: ^3.0.0-nullsafety.1.
3. Add dev dependency flutter_driver: sdk: flutter
4. Run flutter pub get

**Results:** 
```
Because every version of flutter_driver from sdk depends on file 6.0.0 and flutter_cache_manager >=3.0.0-nullsafety.0 depends on file ^6.1.0, flutter_driver from sdk is incompatible with flutter_cache_manager >=3.0.0-nullsafety.0.
So, because {projectName} depends on both flutter_cache_manager ^3.0.0-nullsafety.0 and flutter_driver any from sdk, version solving failed.
pub get failed (1; So, because {projectName} depends on both flutter_cache_manager ^3.0.0-nullsafety.0 and flutter_driver any from sdk, version solving failed.)
```

**Comment by daniil-shumko** [comment]
I have the same problem. And a temporary workaround would be to add `dependency_overrides:` to your `pubspec.yaml` file like so:
```
dependency_overrides:
  file: ^6.1.0
```

**Comment by sidrao2006** [comment]
This issue was solved in [`v3.0.1`](https://pub.dev/packages/flutter_cache_manager/versions/3.0.1) by loosening the dependency constraints for `file`
